settings.php

```diff
@@ -8,11 +8,23 @@
  * Database configuration:
  *
  * Most sites can configure their database by entering the connection string
- * below. If using master/slave databases or multiple connections, see the
+ * below. If using primary/replica databases or multiple connections, see the
```

Comment:
Better use "Primary/Secondary" to use the same terms on the documentation page mentioned below?